### PR TITLE
Add mobile options & compatibility to Header Options

### PR DESCRIPTION
**Pull request by dlmarquis**
I want you all to know that I feel personally victimized by Tumblr's coding decisions, and would like to apologize for the sins I have committed herein.

### CODE REVIEW

File: Extensions/classic_header.js [hunk 1]
```diff
@@ -92,6 +112,17 @@ XKit.extensions.classic_header = new Object({
 			XKit.tools.add_css(" .tab_notice_value { color: #ffffff !important; } .selected .tab_notice, .tab_notice { background: #bc3333 !important; } .tab_bar .tab.selected .tab_anchor, .tab_bar .tab.active .tab_anchor {opacity: 0.5;}", "classic_header_fixed_color");
 		}
 
+		if (XKit.extensions.classic_header.preferences.mobile_sticky.value === true) {
```
Comment (0xazure) [inline Extensions/classic_header.js]: Were we not doing away with the `=== true/false` checks and just leaving it as `value`?

File: Extensions/classic_header.js [hunk 2]
```diff
@@ -92,6 +112,17 @@ XKit.extensions.classic_header = new Object({
...
+			// The nav menu is written terribly. Thanks @staff.
+			XKit.tools.add_css(" #container { position: absolute; top:44px; } .mobile-nav { position: fixed; top: 0; z-index: 99; left: 0; width: calc(100% + 1px); } .nav-menu .drawer, .nav-menu.active .sneeze-guard { height: calc(100vh - 44px); top:44px; }","classic_header_mobile_sticky");
+			$('#footer').insertAfter($('#load_more_posts'));
+		}	
+
+		if (XKit.extensions.classic_header.preferences.mobile_logout.value === true) {
```
Comment (0xazure) [inline Extensions/classic_header.js]: `=== true/false` vs `value`?

File: Extensions/classic_header.js (hunk 1 above)
Comment (dlmarquis) [inline Extensions/classic_header.js]: You're right. I forgot because we apparently missed this extension when we fixed those.

**Comment by dlmarquis** [comment]
I've fixed the if statements now.

File: Extensions/classic_header.js [hunk 3]
```diff
@@ -187,6 +218,8 @@ XKit.extensions.classic_header = new Object({
 		XKit.tools.remove_css("classic_header_fixed_color");
 		XKit.tools.remove_css("classic_header_fixed_position");
 		XKit.tools.remove_css("classic_header_fixed_width");
+		XKit.tools.remove_css("classic_header_mobile_sticky");
+		$(".nav-item-goodbye").remove();
```
Comment (hobinjk) [inline Extensions/classic_header.js]: This should only occur when XKit manually added the logout button:
```javascript
if (XKit.extensions.classic_header.preferences.mobile_logout.value) {
  $(".nav-item-goodbye").remove();
}
```

**Comment by hobinjk** [comment]
Tested locally, LGTM.
Would r+ if not for the merge conflicts.

**Comment by Wolvan** [comment]
@homu r=hobinjk

**Pull request merged**